modules/ROOT/pages/plugins/premium/advtable.adoc

```diff
@@ -6,12 +6,17 @@
 :pluginname: Advanced Tables
 :plugincode: advtable
 :plugincategory: premium
-include::partial$misc/requires_5_1v.adoc[] +
+
+include::partial$misc/requires_5_1v.adoc[]
+
 include::partial$misc/premiumplugin.adoc[]
 
-The `advtable` plugin is a premium plugin that extends the core link:{baseurl}/plugins/opensource/table/[`table` plugin] by adding the following advanced functionality:
-- Sort options for rows and columns.
-- Row numbering column for tables. include::partial$misc/requires_5_9v.adoc[]
+The `advtable` plugin is a premium plugin that extends the core xref:plugins/opensource/table.adoc[`table` plugin] by adding the following advanced functionality:
+
+* Sort options for rows and columns.
+* Row numbering column for tables.
+
+include::partial$misc/requires_5_9v.adoc[]
```

Comment:
```suggestion
* Row numbering column for tables.
include::partial$misc/requires_5_9v.adoc[]
```
This include it meant for the last list item only, so it needs to be directly after it otherwise it'll render outside the list.

Reply:
When I use your suggestion it renders like this:

<img width="783" alt="Screen Shot 2022-03-23 at 23 25 04" src="https://user-images.githubusercontent.com/38878982/159806253-caed5d04-d89d-4912-b72d-789d60f7867d.png">

But when I leave that empty line it renders like this:

<img width="827" alt="Screen Shot 2022-03-23 at 23 26 44" src="https://user-images.githubusercontent.com/38878982/159806417-08462c6d-e0b6-4ebf-870c-b1b1266e891f.png">

What should I do ?

Reply:
There's no way to have an "inline" admonition so this one we definitely want to be rendered as plain text. It's also like that on the current site: https://www.tiny.cloud/docs/plugins/premium/advtable/